assets/js/googlesitekit/widgets/datastore/widgets.js

```diff
@@ -316,6 +317,49 @@ export const selectors = {
 	getWidgetState( state, slug ) {
 		return state.widgetStates[ slug ] || null;
 	},
+
+	/**
+	 * Returns whether or not a widget with a given slug is active.
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @param {Object} state Data store's state.
+	 * @param {string} slug  Widget slug.
+	 * @return {boolean} True for active.
+	 */
+	isWidgetActive( state, slug ) {
+		return selectors.getWidgetState( state, slug ) === null ||
+			selectors.getWidgetState( state, slug ).Component !== Null;
+	},
```

Comment:
While this selector might be useful, I think for our usage in the component where we have to compare `widgets` with `widgetStates` inside utility functions, it would be more beneficial to outsource the low-level logic to another utility function, e.g. `isInactiveWidgetState( widgetState ) => widgetState && widgetState.Component === Null`. This we could then use in e.g. `getWidgetLayout` (see above comment).

Reply:
Updated and removed the selector in favour of the util function in https://github.com/google/site-kit-wp/pull/2731/commits/f8c6a5f4ef31bfb58d58071b4f1c0d6f911be951